### PR TITLE
DO-846 Add SignUp/Login buttons

### PR DESCRIPTION
**Pull request by Litsyshyn**
Added Login/SignUp buttons to landing page.
Added reversed color button (Secondary) for landing page, with same propeties as Primary desighned button, acceps for borders.
Also added .svg image of secondary button, for future.

upd. noticed mistake in button filename, was corrected

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,9 +25,12 @@
...
                     </a>
-                    <a class="navigation__item btn-primary" href="#contact-form">
-                        <span lang="uk">Напишіть нам</span>
-                        <span lang="en">Contact us</span>
+                    <a class="navigation__item btn-secondary" href="" target="_blank">
+                        <span lang="uk">Увійти</span>
+                        <span lang="en">Login</span>
+                    <a class="navigation__item btn-primary" href="" target="_blank">
+                        <span lang="uk">Зареєструватись</span>
+                        <span lang="en">Sign up</span>
```
Comment (roman-tiukh) [inline index.html:33]: add temporary links to login and sign up pages on platform

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Litsyshyn) [reply index.html:33]: +

**Comment by Litsyshyn** [comment]
Actually i was add them at the begining, but decide to delete until functional will be available )

**Review by roman-tiukh — APPROVED** [review]

**Comment by Litsyshyn** [comment]
Ooops, sorry @roman-tiukh

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**